src/cargo/util/command_prelude.rs

```diff
@@ -295,6 +297,15 @@ pub trait CommandExt: Sized {
         )
     }
 
+    fn arg_lockfile_path(self) -> Self {
+        self._arg(
+            opt("lockfile-path", "Path to the Cargo.lock file (unstable)")
+                .value_name("FILE")
+                // TODO: seemed to be the best option, but maybe should be something else?
```

Comment:
I feel it makes sense to put `--manifest-path` and `--lockfile-path` side by side.

```suggestion
```